## rename-genbox-man.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: rename genbox in manpages.

@DPATCH@
--- a/doc/man/cat1/genblinds.1
+++ b/doc/man/cat1/genblinds.1
@@ -39,7 +39,7 @@ AAUUTTHHOORR
        Jean‐Louis Scartezzini and Greg Ward
 
 SSEEEE AALLSSOO
-       genbox(1),  genrev(1),  gensurf(1),   genworm(1),   rpict(1),   rvu(1),
+       genrbox(1),  genrev(1),  gensurf(1),   genworm(1),   rpict(1),   rvu(1),
        xform(1)
 
 
--- a/doc/man/cat1/genbox.1
+++ b/doc/man/cat1/genbox.1
@@ -1,9 +1,9 @@
-GENBOX(1)                                                            GENBOX(1)
+GENRBOX(1)                                                            GENRBOX(1)
 
 
 
 NNAAMMEE
-       genbox ‐ generate a RADIANCE description of a box
+       genrbox ‐ generate a RADIANCE description of a box
 
 SSYYNNOOPPSSIISS
        ggeennbbooxx mmaatt nnaammee xxssiizz yyssiizz zzssiizz [ −−ii ][ −−rr rraadd | −−bb bbeevv ]
@@ -21,7 +21,7 @@ DDEESSCCRRIIPPTTIIOONN
 EEXXAAMMPPLLEE
        To produce a rectangular box made of wood with beveled edges:
 
-         genbox wood box1 5 8 3 −b .5 > box1
+         genrbox wood box1 5 8 3 −b .5 > box1
 
 AAUUTTHHOORR
        Greg Ward
@@ -35,4 +35,4 @@ SSEEEE AALLSSOO
 
 
 
-RADIANCE                           11/15/93                          GENBOX(1)
+RADIANCE                           11/15/93                          GENRBOX(1)
--- a/doc/man/cat1/genclock.1
+++ b/doc/man/cat1/genclock.1
@@ -37,7 +37,7 @@ AAUUTTHHOORR
        Greg Ward
 
 SSEEEE AALLSSOO
-       genbox(1),  genrev(1),  gensurf(1),   genworm(1),   rpict(1),   rvu(1),
+       genrbox(1),  genrev(1),  gensurf(1),   genworm(1),   rpict(1),   rvu(1),
        xform(1)
 
 
--- a/doc/man/cat1/genprism.1
+++ b/doc/man/cat1/genprism.1
@@ -35,7 +35,7 @@ BBUUGGSS
        normals.  If the normals face inward, the appearance will be bizarre.
 
 SSEEEE AALLSSOO
-       genbox(1),   genrev(1),   gensurf(1),   genworm(1),  rpict(1),  rvu(1),
+       genrbox(1),   genrev(1),   gensurf(1),   genworm(1),  rpict(1),  rvu(1),
        xform(1)
 
 
--- a/doc/man/cat1/genrev.1
+++ b/doc/man/cat1/genrev.1
@@ -33,7 +33,7 @@ BBUUGGSS
        site side.
 
 SSEEEE AALLSSOO
-       genbox(1), gensurf(1), genworm(1), icalc(1), rpict(1), rvu(1), xform(1)
+       genrbox(1), gensurf(1), genworm(1), icalc(1), rpict(1), rvu(1), xform(1)
 
 
 
--- a/doc/man/cat1/gensurf.1
+++ b/doc/man/cat1/gensurf.1
@@ -68,7 +68,7 @@ BBUUGGSS
        [0,1] boundaries of s and t.
 
 SSEEEE AALLSSOO
-       genbox(1), genrev(1), genworm(1),  icalc(1),  obj2mesh(1),  obj2rad(1),
+       genrbox(1), genrev(1), genworm(1),  icalc(1),  obj2mesh(1),  obj2rad(1),
        rpict(1), rvu(1), xform(1)
 
 
--- a/doc/man/cat1/genworm.1
+++ b/doc/man/cat1/genworm.1
@@ -31,7 +31,7 @@ BBUUGGSS
        back inside itself without making a mess.
 
 SSEEEE AALLSSOO
-       genbox(1), genrev(1), gensurf(1), icalc(1), rpict(1), rvu(1), xform(1)
+       genrbox(1), genrev(1), gensurf(1), icalc(1), rpict(1), rvu(1), xform(1)
 
 
 
--- a/doc/man/cat1/xform.1
+++ b/doc/man/cat1/xform.1
@@ -111,7 +111,7 @@ BBUUGGSS
        Only regular (distortion‐free) transformations are allowed.
 
 SSEEEE AALLSSOO
-       genbox(1), gensurf(1), oconv(1), replmarks(1), rpict(1), rvu(1)
+       genrbox(1), gensurf(1), oconv(1), replmarks(1), rpict(1), rvu(1)
 
 
 
--- a/doc/man/man1/genblinds.1
+++ b/doc/man/man1/genblinds.1
@@ -57,4 +57,4 @@ genblinds white blind 1 46 88 25 15 \-r
 .SH AUTHOR
 Jean-Louis Scartezzini and Greg Ward
 .SH "SEE ALSO"
-genbox(1), genrev(1), gensurf(1), genworm(1), rpict(1), rvu(1), xform(1)
+genrbox(1), genrev(1), gensurf(1), genworm(1), rpict(1), rvu(1), xform(1)
--- a/doc/man/man1/genbox.1
+++ b/doc/man/man1/genbox.1
@@ -1,9 +1,9 @@
-.\" RCSid "$Id: genbox.1,v 1.4 2007/09/04 17:36:40 greg Exp $"
+.\" RCSid "$Id: genrbox.1,v 1.4 2007/09/04 17:36:40 greg Exp $"
 .TH GENBOX 1 11/15/93 RADIANCE
 .SH NAME
-genbox - generate a RADIANCE description of a box
+genrbox - generate a RADIANCE description of a box
 .SH SYNOPSIS
-.B "genbox mat name xsiz ysiz zsiz"
+.B "genrbox mat name xsiz ysiz zsiz"
 [
 .B \-i
 ][
@@ -35,7 +35,7 @@ option can be used to specify the indent
 .SH EXAMPLE
 To produce a rectangular box made of wood with beveled edges:
 .IP "" .2i
-genbox wood box1 5 8 3 \-b .5 > box1
+genrbox wood box1 5 8 3 \-b .5 > box1
 .SH AUTHOR
 Greg Ward
 .SH BUGS
--- a/doc/man/man1/genclock.1
+++ b/doc/man/man1/genclock.1
@@ -54,4 +54,4 @@ genclock 10:35 | xform \-s 6 \-rz 90 \-t
 .SH AUTHOR
 Greg Ward
 .SH "SEE ALSO"
-genbox(1), genrev(1), gensurf(1), genworm(1), rpict(1), rvu(1), xform(1)
+genrbox(1), genrev(1), gensurf(1), genworm(1), rpict(1), rvu(1), xform(1)
--- a/doc/man/man1/genprism.1
+++ b/doc/man/man1/genprism.1
@@ -57,4 +57,4 @@ The rounding option only works for opaqu
 normals.
 If the normals face inward, the appearance will be bizarre.
 .SH "SEE ALSO"
-genbox(1), genrev(1), gensurf(1), genworm(1), rpict(1), rvu(1), xform(1)
+genrbox(1), genrev(1), gensurf(1), genworm(1), rpict(1), rvu(1), xform(1)
--- a/doc/man/man1/genrev.1
+++ b/doc/man/man1/genrev.1
@@ -57,4 +57,4 @@ Greg Ward
 The \-s option doesn't modify the surface normal correctly
 for the opposite side.
 .SH "SEE ALSO"
-genbox(1), gensurf(1), genworm(1), icalc(1), rpict(1), rvu(1), xform(1)
+genrbox(1), gensurf(1), genworm(1), icalc(1), rpict(1), rvu(1), xform(1)
--- a/doc/man/man1/gensurf.1
+++ b/doc/man/man1/gensurf.1
@@ -115,5 +115,5 @@ Greg Ward
 The smoothing operation requires that functions be defined
 beyond the [0,1] boundaries of s and t.
 .SH "SEE ALSO"
-genbox(1), genrev(1), genworm(1), icalc(1),
+genrbox(1), genrev(1), genworm(1), icalc(1),
 obj2mesh(1), obj2rad(1), rpict(1), rvu(1), xform(1)
--- a/doc/man/man1/genworm.1
+++ b/doc/man/man1/genworm.1
@@ -46,4 +46,4 @@ Since the worm is constructed of interse
 opaque materials should be used with this object.
 Also, a worm cannot double back inside itself without making a mess.
 .SH "SEE ALSO"
-genbox(1), genrev(1), gensurf(1), icalc(1), rpict(1), rvu(1), xform(1)
+genrbox(1), genrev(1), gensurf(1), icalc(1), rpict(1), rvu(1), xform(1)
--- a/doc/man/man1/xform.1
+++ b/doc/man/man1/xform.1
@@ -184,4 +184,4 @@ Greg Ward
 .SH BUGS
 Only regular (distortion-free) transformations are allowed.
 .SH "SEE ALSO"
-genbox(1), gensurf(1), oconv(1), replmarks(1), rpict(1), rvu(1)
+genrbox(1), gensurf(1), oconv(1), replmarks(1), rpict(1), rvu(1)
